.travis.yml

```diff
@@ -38,14 +43,12 @@ jobs:
         - pip install --user --upgrade pip
         - pip install --user yamllint
         # install golangci-lint
-        # workaround for https://github.com/golangci/golangci-lint/issues/575
         - curl -sf
-          "https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh"
+          "https://install.goreleaser.com/github.com/golangci/golangci-lint.sh"
           | bash -s -- -b $GOPATH/bin "${GOLANGCI_VERSION}"
       script:
-        - scripts/lint-text.sh --require-all
-        - scripts/lint-go.sh
-        - scripts/test-go.sh
+        - make static-check
+        - make dep-check || travis_terminate 1;
```

Comment:
We are not running `scripts/test-go.sh` anymore or is that an inadvertent change?

Reply:
missed it. added  `make go-test`